plugins/org.jkiss.dbeaver.data.gis.view/src/org/jkiss/dbeaver/ui/gis/panel/GISLeafletViewer.java

```diff
@@ -109,36 +110,50 @@ public GISLeafletViewer(Composite parent, @NotNull DBDAttributeBinding[] binding
         composite = UIUtils.createPlaceholder(parent, 1);
         CSSUtils.setCSSClass(composite, DBStyles.COLORED_BY_CONNECTION_TYPE);
 
-        browser = new Browser(composite, SWT.NONE);
-        browser.setLayoutData(new GridData(GridData.FILL_BOTH));
-
-        new BrowserFunction(browser, "setClipboardContents") {
-            @Override
-            public Object function(Object[] arguments) {
-                UIUtils.setClipboardContents(Display.getCurrent(), TextTransfer.getInstance(), arguments[0]);
-                return null;
+        Browser newBrowser;
+        try {
+            newBrowser = new Browser(composite, SWT.NONE);
+        } catch (SWTError error) {
+            for (Control control : composite.getChildren()) {
+                control.dispose();
+            }
```

Comment:
What's the purpose of this code in this particular place?

Reply:
The creation is started, but then something happened. Because of this we have not fully initialized Browser control - it is shown like a small white area in the top with the data from the previously rendered value panel state. That's not good at all.

Reply:
What does it look like now then?

Reply:
There is grey area in the value panel with the text `No editor for [" + previewController.getValueType().getTypeName() + "]"`

Reply:
We already had this when valueEditor is null. It looks like this 
![image](https://user-images.githubusercontent.com/28875055/188845551-4599320e-2d1d-4e1a-90ce-ac59e490d767.png)

Reply:
It looks like without dispose call, because we added Browser control to the composite, but then the error occurred and Browser control can't be initialized and used.
![image](https://user-images.githubusercontent.com/28875055/188848107-597a2ff0-584a-43f9-bbfc-5ffa193e6bba.png)

Reply:
And the text in the white field is just the text which was previously rendered in the value field. It may be another field value, or in some special cases some strange symbols.

Reply:
Well, okay. Thanks.